checkbox/checkbox-provider-npu/units/jobs.pxu

```diff
@@ -264,26 +309,29 @@ command:
 id: umd/CommandCopyPerf
 category_id: npu
 flags: simple
+depends: kmd/CharDevicePermissions
 requires: executable.name == 'intel-npu-driver.vpu-umd-test'
 _summary: CommandCopyPerf
 estimated_duration: 2s
 command:
   cd $HOME/snap/intel-npu-driver/current
   intel-npu-driver.vpu-umd-test --gtest_filter=CommandCopyPerf.* --config=basic.yaml
 
-id: umd/SystemToSystem
+id: umd/SystemToSystem/CommandCopyFlag
```

Comment:
Small note: I noticed none of the `SystemToSystem` tests were running because the filter I was using was incorrect.